src/code/PreRender.c

```diff
@@ -1,42 +1,615 @@
+            { gDPSetTextureImage(gfx++, 4, 1, this->width, this->cvgSave); do { { Gfx *_g = (Gfx *) (gfx++); _g->words.w0 = (((((unsigned int) ((((unsigned int) 0xf5) & ((0x01 << 8) - 1)) << 24)) | ((unsigned int) ((((unsigned int) 4) & ((0x01 << 3) - 1)) << 21))) | ((unsigned int) ((((unsigned int) new_var) & ((0x01 << 2) - new_var)) << 19))) | ((unsigned int) ((((unsigned int) (((((lrx - uls) + new_var) * new_var) + 7) >> 3)) & ((0x01 << 9) - new_var)) << 9))) | ((unsigned int) ((((unsigned int) 0x0160) & ((0x01 << 9) - new_var)) << 0)); _g->words.w1 = ((((((((unsigned int) ((((unsigned int) 7) & ((0x01 << 3) - new_var)) << 24)) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 20))) | ((unsigned int) ((((unsigned int) (0 | 0)) & ((0x01 << 2) - new_var)) << 18))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 14))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 10))) | ((unsigned int) ((((unsigned int) (0 | 0)) & ((0x01 << 2) - new_var)) << 8))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 4))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 0)); } } while (0); gDPLoadSync(gfx++); gDPLoadTile( gfx++, G_TX_LOADTILE, (uls)<<G_TEXTURE_IMAGE_FRAC, (ult)<<G_TEXTURE_IMAGE_FRAC, (lrx)<<G_TEXTURE_IMAGE_FRAC, (lry)<<G_TEXTURE_IMAGE_FRAC); gDPPipeSync(gfx++); do { { Gfx *_g = (Gfx *) (gfx++); _g->words.w0 = (((((unsigned int) ((((unsigned int) 0xf5) & ((0x01 << 8) - new_var)) << 24)) | ((unsigned int) ((((unsigned int) 4) & ((0x01 << 3) - new_var)) << 21))) | ((unsigned int) ((((unsigned int) new_var) & ((0x01 << 2) - new_var)) << 19))) | ((unsigned int) ((((unsigned int) (((((lrx - uls) + new_var) * new_var) + 7) >> 3)) & ((0x01 << 9) - new_var)) << 9))) | ((unsigned int) ((((unsigned int) 0x0160) & ((0x01 << 9) - new_var)) << 0)); _g->words.w1 = ((((((((unsigned int) ((((unsigned int) new_var) & ((0x01 << 3) - new_var)) << 24)) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 20))) | ((unsigned int) ((((unsigned int) (0 | 0)) & ((0x01 << 2) - new_var)) << 18))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 14))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - new_var)) << 10))) | ((unsigned int) ((((unsigned int) (0 | 0)) & ((0x01 << 2) - 1)) << 8))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - 1)) << 4))) | ((unsigned int) ((((unsigned int) 0) & ((0x01 << 4) - 1)) << 0)); } } while (0); gDPSetTileSize(gfx++, new_var, (uls)<<G_TEXTURE_IMAGE_FRAC, (ult)<<G_TEXTURE_IMAGE_FRAC, (lrx)<<G_TEXTURE_IMAGE_FRAC, (lry)<<G_TEXTURE_IMAGE_FRAC); } ;
+        }
+        #endif
```

Comment:
ive never seen us do a non matching inside a func like this. I guess it makes sense if youve isolated the problem, but seems weird

Reply:
@Roman971 mentioned in discord we should probably not have non matchings in the middle of a function and I agree.
If you want to comment exactly what the issue is and what that fix is, feel free to do that tho

Reply:
@Random06457 not sure if the above was missed, but i think this is all thats holding this up. id just put the whole function in a non matching instead of this part